regression/CMakeLists.txt

```diff
@@ -26,12 +26,12 @@ endfunction()
 
 # NOTE: In order to make the best of the concurrency set sort the tests from the slowest to fastest.
 if(APPLE)
-    set(REGRESSIONS esbmc-unix esbmc-unix2 esbmc cbmc cstd llvm floats floats-regression k-induction k-induction-parallel nonz3 bitwuzla)
+    set(REGRESSIONS esbmc-unix esbmc-unix2 esbmc cbmc cstd llvm floats floats-regression k-induction k-induction-parallel nonz3 bitwuzla incremental)
 elseif(WIN32)
     # FUTURE: Add floats-regression esbmc-cpp/cpp
     set(REGRESSIONS esbmc cbmc cstd llvm floats  k-induction )
 else()
-    set(REGRESSIONS esbmc-unix esbmc-unix2 esbmc cbmc cstd llvm floats floats-regression k-induction esbmc-cpp/cpp csmith k-induction-parallel nonz3 bitwuzla)
+    set(REGRESSIONS esbmc-unix esbmc-unix2 esbmc cbmc cstd llvm floats floats-regression k-induction esbmc-cpp/cpp csmith k-induction-parallel nonz3 bitwuzla incremental)
```

Comment:
Here incremental means incremental SMT solving, right? Could you rename? Incremental will probably be confused with incremental-bmc for newer test cases.

Reply:
Yes. I have renamed the folder to `incremental-smt`.